docs/pages/Kratos/Expressions/General/Visualization.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Visualization
+keywords:
+tags: [visualization expression vtu vtk]
+sidebar: kratos_expressions
+summary:
+---
+
+## Introduction
+
+Expressions can be written to the `Vtu` format and can be easily visualized using any compatible software such as [Paraview](https://www.paraview.org/).
+
+This output is compatible with **shared memory** and **distributed memory** architectures. In **shared memory** architectures, it will write only one file having the extension `*.vtu`. In **distributed memory** architectures, it will write one `*.vtu` file per rank, and then one `*.pvtu` for all ranks linking all the `*.vtu` files from all ranks.
```

Comment:
```suggestion
This output is compatible with **shared memory** and **distributed memory** parallelization. In **shared memory** parallelization, it will write only one file having the extension `*.vtu`. In **distributed memory** parallelization, it will write one `*.vtu` file per rank, and then one `*.pvtu` for all ranks linking all the `*.vtu` files from all ranks.
```